src/main/java/oc/Eintrag.java

```diff
@@ -285,7 +285,7 @@ public String getText() {  // der Text, der in der Textarea als Armeeliste angez
 
         //Bei alle Punktkosten: hier wird bei kostenlosen Einträgen, bei denen nur die erste Option gewählt ist, welche selbst aber noch Kosten und weitere eigene Optionen hat, die Gesamtpunktzahl nicht angezeigt. Deswegen: "gesamtpunkteImmerAnzeigen" hinzugefügt.
         if (this.getKosten() != 0 && (cnt > 1 || !BuildaHQ.allePunktkosten || (optionen.size() > 0 ? optionen.elementAt(0).getKosten() != this.getKosten() : false) || gesamtpunkteImmerAnzeigen)) {
-            text.append(" " + BuildaHQ.formatierItalic(" - - - > " + entferneNullNachkomma(getKosten()) + " " + BuildaHQ.translate(OnlineCodex.getGame() == OnlineCodex.NECROMUNDA ? "Credits" : "Punkte")));
+            text.append(" " + BuildaHQ.formatierItalic(" - - - > " + entferneNullNachkomma(getKosten()) + " " + BuildaHQ.translate(OnlineCodex.getGame() == OnlineCodex.NECROMUNDA ? "Credits" : "Punkte") + " (PL " + getPower()+ ")"));
```

Comment:
Maybe use `String.format()` instead of a super long line?

Something along the lines of:

```java
text.append(String.format(" [I] - - - > %d [/I] %s (PL %d)",
        getKosten(),
        BuildaHQ.translate("Punkte"),
        getPower()));
```